src/app/contexts/RequestContext/getStatsDestination/index.js

```diff
@@ -1,12 +1,19 @@
 const getStatsDestination = ({ isUK = true, env = 'test', service }) => {
   let destination = '';
-
-  if (service === 'news') {
-    destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
-  } else {
-    destination = 'WS_NEWS_LANGUAGES';
+  switch (service) {
+    case 'news':
+      destination = isUK !== false ? 'NEWS_PS' : 'NEWS_GNL';
```

Comment:
This is really hard to read, can you comment what the logic of the nested conditions are.

Reply:
can this just be

```js
destination = isUK ? 'NEWS_PS' : 'NEWS_GNL';
```